core/new-gui/src/app/workspace/component/operator-panel/operator-label/operator-label.component.spec.ts

```diff
@@ -67,6 +72,63 @@ describe('OperatorLabelComponent', () => {
     expect(jqueryElement.data('uiDraggable')).toBeTruthy();
   });
 
+  it('should call the mouseEnter function once the cursor is hovering above a operator label', () => {
+    const spy = spyOn<any>(component, 'mouseEnter');
+    const operatorLabelElement = fixture.debugElement.query(By.css('#' + component.operatorLabelID));
+    operatorLabelElement.triggerEventHandler('mouseenter', component);
+    expect(spy).toHaveBeenCalled();
+  });
+
+  it('should call the mouseLeave function once the cursor leaves a operator label', () => {
+    const spy = spyOn<any>(component, 'mouseLeave');
+    const operatorLabelElement = fixture.debugElement.query(By.css('#' + component.operatorLabelID));
+    operatorLabelElement.triggerEventHandler('mouseleave', component);
+    expect(spy).toHaveBeenCalled();
+  });
+
+  it('should emits a command to open an tooltip instance after 500ms delay', marbles((m) => {
+    const expectedStream = m.hot('500ms -a-');
+    const actualStream = component.getopenCommandsObservable().map(() => 'a');
+    m.hot('-a-').do(() => component.mouseEnter()).subscribe();
+    m.expect(actualStream).toBeObservable(expectedStream);
+  }));
+
+  it('should display a tooltip instance with the correct content when openCommandObservable$ emits a value', marbles((m) => {
+    const operatorLabelElement = fixture.debugElement.query(By.css('#' + component.operatorLabelID));
+    m.hot('-a-').do(() => component.mouseEnter()).subscribe();
+    component.getopenCommandsObservable().subscribe(x => {
+      if (operatorLabelElement.parent) {
+        const tooltipInstance = operatorLabelElement.parent.childNodes[1].nativeNode;
+        expect(tooltipInstance.innerText).toBe(mockOperatorData.additionalMetadata.operatorDescription);
+      }
```

Comment:
The subscription `component.getopenCommandsObservable().subscribe(x => ...` should be moved in front of `m.hot()....` to prevent the case which the marble is executed before this subscription is set up.